### ISSUE TITLE
Please provide a npm package for nuxt SSR production deploy that has only runtime dependencies

### ISSUE THREAD

**Issue by far-blue** [nuxt/nuxt.js#5879]
### What problem does this feature solve?

At the moment, the nuxt npm package has dependencies such as @nuxt/webpack that pull in webpack, babel, postcss, node-sass etc. While this is fine in a dev or build environment it significantly increases the deploy size and security surface of a production SSR application.

### What does the proposed changes look like?

Rather than the current nuxt package, 2 packages would be available - one with the build / dev dependencies and one without. Kind of line `micro` and `micro-dev`. I suggest the full-fat version continue to be `nuxt` while the new one could be `@nuxt/server-runtime` or something.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9315">#c9315</a>)</em></sub></div>

**Comment by pimlie** [comment]
Do you mean https://www.npmjs.com/package/nuxt-start ?

**Comment by far-blue** [comment]
Yes, great to see it exists! Although I failed to find it via searching google or by reading the Nuxt docs so may I request that its existence is mentioned somewhere?

A mention here would be great: https://nuxtjs.org/guide/commands#server-side-rendered-deployment-universal-ssr-

:)

**Comment by pimlie** [comment]
Hmm yes, strange its not mentioned.

It would be really great if you could add a PR for that in the docs: https://github.com/nuxt/docs :)
